apps/src/templates/projects/ProjectCard.story.jsx

```diff
@@ -12,33 +12,58 @@ const defaultData = {
   publishedToClass: false
 };
 
-const PUBLISHED_PUBLIC = {
+const PUBLISHED_BOTH = {
   ...defaultData,
+  publishedToPublic: true,
+  publishedToClass: true,
+};
+
+const personalData = {
+  channel: 'abcdef',
+  name: 'Puppy Playdate',
+  type: 'applab',
+  updatedAt: '2016-12-31T23:59:59.999-08:00',
+  publishedToPublic: false,
+  publishedToClass: false
+};
+
+const publishedToPublicPersonal = {
+  ...personalData,
   publishedToPublic: true
 };
 
-const PUBLISHED_CLASS = {
-  ...defaultData,
+const publishedToClassPersonal = {
```

Comment:
I'm confused about why some of these const are CAPS_SNAKE and some are lowerCamel. Should they be consistent?